cypress-public.json

```diff
@@ -5,6 +5,7 @@
   "reporterOptions": {
     "toConsole": true
   },
+  "video": false,
```

Comment:
Another possible solution identified in the Cypress issue is to do...  That way, we still (possibly) get the video on a failure.

```json
"video": true,
"videoUploadOnPasses": false,
```